daemon/daemon_unix.go

```diff
@@ -1151,3 +1154,20 @@ func setupOOMScoreAdj(score int) error {
 	f.Close()
 	return err
 }
+
+func (daemon *Daemon) setupSeccompProfile() error {
+	b, err := ioutil.ReadFile(seccompProfilePath)
+	if err != nil {
+		// TODO(runcom): generate a default seccomp profile if system wide isn't present?
```

Comment:
I'm not sure we're going to override the default with a flag in the daemon given people/admin/distros can already override the default at /etc/docker/seccomp.json 

I believe this just the intention of @justincormack ad well.